modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/exception/mapper/ExceptionMapper.java

```diff
@@ -19,13 +19,30 @@
 import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
 import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
 
+import javax.ws.rs.core.Context;
 import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.Providers;
 
 /**
  * @author Javier Gamarra
  */
 public class ExceptionMapper extends BaseExceptionMapper<Exception> {
 
+	@Override
+	public Response toResponse(Exception exception) {
+		Throwable throwable = exception.getCause();
+
+		javax.ws.rs.ext.ExceptionMapper<Throwable> exceptionMapper =
+			_providers.getExceptionMapper(
+				(Class<Throwable>)throwable.getClass());
```

Comment:
Don't like at all this cast but it is the only way that I find to implement working code that passes the Source Formatter